apidocs/cloud-api-source/source/includes/container.md

```diff
@@ -420,7 +422,7 @@ Available in Docker Cloud's **REST API**
 
 ### HTTP Request
 
-`GET /api/app/v1/container/`
+`GET /api/app/v1/(optional_namespace)/container/`
```

Comment:
Updated: now using `GET /api/app/v1/[optional_namespace/]container/`
Removed the `( )` because they're performing the same function as the `[ ]`. Keeping the word `optional` since that's an extra cue for folks who may not be aware of the bracket convention.